### DISCUSSION TITLE
can i configure mobileproxy to use outline server ?

### DISCUSSION THREAD

**Discussion by namdroid** [OutlineFoundation/outline-sdk#340] [Q&A] [answered]
I have integrated mobile proxy to our android app. It seems to work. But can i add specific outline server to the mobile proxy ? If the app connect through other http2transport proxy, can this proxy blocked by other ?

**Comment by fortuna** [comment, marked as answer]
Yes, you can use a Static configuration with a ss:// link to connect to an Outline Server:
https://github.com/Jigsaw-Code/outline-sdk/tree/main/x/mobileproxy#using-static-transport-configuration

You'd create a dialer like

```kotlin
val dialer = StreamDialer("ss://CIPHER-INFO@example.com:4321")
```
